src/heifimage.cpp

```diff
@@ -0,0 +1,76 @@
+#define DEBUG
+
+#include "config.h"
+
+#include "heifimage.hpp"
+#include "image_int.hpp"
+#include "enforce.hpp"
+#include "futils.hpp"
+#include "basicio.hpp"
+#include "tags.hpp"
+#include "tags_int.hpp"
+#include "types.hpp"
+#include "convert.hpp"
+
+#include <cmath>
```

Comment:
It looks like you copy/pasted these headers from other files. Could you please drop the ones that are not needed?